pages/api/user/avatar.ts

```diff
@@ -0,0 +1,20 @@
+// import fs from "fs";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { getSession } from "@lib/auth";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const session = await getSession({ req: req });
+
+  if (!session) {
+    res.status(401).json({ message: "Not authenticated" });
+    return;
+  }
+
+  const decoded = req.body.avatar.toString().replace("data:image/png;base64,", "");
```

Comment:
This method allows for accounting for image/jpeg if you use the value here to go into the header :)

Reply:
Yeah, good call!

Reply:
this resolved?